### PR TITLE
Feature/support gdpr compliance regulation

### PR DESCRIPTION
**Pull request by oussamaessaji**

### CODE REVIEW

File: MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
   "categories": {
     "OWASP_MASVS_L2": [
       "MSTG_AUTH_8"
+    ],
```
Comment (3asm) [inline MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json:15]: You first need to add the rules to RE. Add them with a key and mention the key here.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json [hunk 2]
```diff
@@ -12,6 +12,10 @@
...
+    "GDPR_Chapter_IV": [
+      "Article 32 - Security of processing",
+      "Article 25 - protection by design and by default"
```
Comment (amine3) [inline MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json:18]: You should add the GDPR category in scan_categories in RE and just mention the key here.

**Review by amine3 — CHANGES_REQUESTED** [review]

**Review by amine3 — APPROVED** [review]

File: MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json [hunk 3]
```diff
@@ -12,6 +12,10 @@
   "categories": {
     "OWASP_MASVS_L2": [
       "MSTG_AUTH_8"
+    ],
+    "GDPR": [
+      "art_5",
+      "art_32"
```
Comment (3asm) [inline MOBILE_CLIENT/ANDROID/_HIGH/APK_BIOMETRIC_AUTHENTICATION_BYPASS/meta.json:18]: Can you change them to all caps.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: MOBILE_CLIENT/IOS/_MEDIUM/IPA_URL_SCHEME_HIJACKING/meta.json [hunk 4]
```diff
@@ -14,6 +14,10 @@
     ],
     "OWASP_MASVS_L3": [
       "MSTG_PLATFORM_3"
+    ],
```
Comment (3asm) [inline MOBILE_CLIENT/IOS/_MEDIUM/IPA_URL_SCHEME_HIJACKING/meta.json:17]: I don't see any mentions of article 24 and 44 in the findings. 44 is related to data transfer. 24 is related to data processing.

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**